src/common/helper.ts

```diff
@@ -177,6 +177,115 @@ function evaluationString(fun: Function | string, ...args: unknown[]): string {
   return `(${fun})(${args.map(serializeArgument).join(',')})`;
 }
 
+function pageBindingInitString(type: string, name: string): string {
+  return evaluationString(addPageBinding, type, name);
```

Comment:
I don't like relying on function hoisting — WDYT about moving the `return` to be the last line in this function body?

Reply:
Done.